components/calendar/README.md

```diff
@@ -0,0 +1,25 @@
+# Calendar
+
+The `d2l-calendar` component can be used to display a responsively sized calendar that allows for date selection. It indicates a currently selected date if a `selected-value` specified, or if the user selects a date.
+
+![Calendar](./screenshots/calendar.png?raw=true)
+
+```html
+<script type="module">
+  import '@brightspace-ui/core/components/calendar/calendar.js';
+</script>
+<!-- Today is February 21, 2020 -->
```

Comment:
It's because in the screenshot Feb 21st has a larger font size and is bold, indicating that it is today, so the comment is specifically calling that out so the user knows that's why the 21st looks like that.